Add interfaces/screen-fold.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/3b5cfdd/ed/idl/screen-fold.idl
Build: https://travis-ci.org/w3c/webref/builds/219573766

stephenmcgruer commented:
Created a draft PR at https://github.com/web-platform-tests/wpt/pull/25761 to add the idlharness, but I suspect nobody implements this yet.

stephenmcgruer reviewed (changes requested):
[Blocking landing]

autofoolip committed:
Add interfaces/screen-fold.idl

Source: https://github.com/w3c/webref/blob/3b5cfdd/ed/idl/screen-fold.idl
Build: https://travis-ci.org/w3c/webref/builds/219573766
